make/cats

```diff
@@ -38,7 +38,20 @@ kubectl create --namespace="${NAMESPACE}" --filename="kube/bosh-task/acceptance-
 
 stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-cats::create end
 
-sleep 5 ; # Slap dash: Wait for container to be up and running
+i=0
+
+set +o errexit
+while [ "$(k get pod cf:^acceptance-tests -o=json | jq -r .status.phase)" != "Running" ]
```

Comment:
Can we assume that the first is a digit ?
This is a bosh-task, and I see no pod number, just the trailing unique hex code.